site/app/templates/Navigation.twig

```diff
@@ -97,17 +97,10 @@
     {% if button is null %}
         <div class="button-placeholder"></div>
     {% else %}
-        <a class="{{ button.getClass() }}{{ (button.getTitle() == 'Edit' or button.getTitle() == 'Delete') and has_build_error ? 'edit-build-error' : '' }} {% if button.hasOnclick() %} key_to_click {% endif %}{% if button.isDisabled() %} disabled{% endif %}" tabindex="0"
+        <a class="{{ button.getClass() }}{{ (button.getTitle() == 'Edit' or button.getTitle() == 'Delete') and has_build_error ? 'edit-build-error' : '' }} {% if button.hasOnclick() %} key_to_click {% endif %}{% if button.isDisabled() %} disabled{% endif %}" tabindex="0" style="pointer-events: auto;"
             {% if button.isDisabled() %}
-                {% if core.getUser().accessFullGrading() %}
-                    {% set str1 = 'Students are expected to satisfactorily complete ' %}
-                    {% set str2 = ' before they may access this gradeable.  Click OK to bypass this requirement.'%}
-                    onclick="confirmBypass('{{ str1 ~ button.getPrerequisite() ~ str2 }}', '{{ button.getHref() }}' );"
```

Comment:
we also lose this functionality due to this PR

Reply:
For these deleted lines of code, Barb specifically wanted this to be done so that both students and instructors see the same exact view and have the same accessibility for a locked gradeable.